ChangeLog

```diff
@@ -1,3 +1,6 @@
+* 0.35.6
+    * Update documentation regarding user setup
+
```

Comment:
I don't think we should release a new version at all.  The changes are 100% in user documentation, so updating package on user's side makes absolutely zero difference.

Reply:
I propose to make doc change with no Python Client release.  We have already released four versions with absolutely zero changes in behaviour.  I am prepared to rebuild the docs manually myself to skip this whole release hassle, so please ditch these version bumps if possible.

Reply:
Ofc you can update documentation manually. As you want. :)

Sorry, but ditch version bumps not possible, it's a preparation for future release process update.

Reply:
Discussed in DM with Oleg: I removed the version bump and Klas can proceed with the manual doc release